src/ontology/cl-edit.owl

```diff
@@ -14196,9 +14197,9 @@ AnnotationAssertion(rdfs:label obo:CL_0002061 "T-helper 9 cell")
 EquivalentClasses(obo:CL_0002061 ObjectIntersectionOf(obo:CL_0000492 ObjectSomeValuesFrom(obo:RO_0002104 obo:PR_000008982) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0002215) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0032638)))
 SubClassOf(Annotation(oboInOwl:is_inferred "true") obo:CL_0002061 obo:CL_0000492)
 
-# Class: obo:CL_0002062 (type I pneumocyte)
+# Class: obo:CL_0002062 (pulmonary alveolar type 1 cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm") Annotation(oboInOwl:hasDbXref "PMID:20054144") Annotation(oboInOwl:hasDbXref "http://www.copewithcytokines.de") obo:IAO_0000115 obo:CL_0002062 "A type I pneumocyte is a flattened, branched pneumocyte that covers more than 98% of the alveolar surface. This large cell has thin (50-100 nm) cytoplasmic extensions to form the air-blood barrier essential for normal gas exchange.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm") Annotation(oboInOwl:hasDbXref "PMID:20054144") Annotation(oboInOwl:hasDbXref "PMID:36138472") Annotation(oboInOwl:hasDbXref "http://www.copewithcytokines.de") obo:IAO_0000115 obo:CL_0002062 "A squamous pulmonary alveolar epithelial cell that is flattened and branched. A pulmonary alveolar type 1 cell covers more than 98% of the alveolar surface. This large cell has thin (50-100 nm) cytoplasmic extensions to form the air-blood barrier essential for normal gas exchange.")
```

Comment:
Hi there, this paper PMID:36138472 refers to it as pulmonary alveolar type 2 cell rather than pulmonary alveolar type1 cell. Could you use a different reference maybe this one PMID: 29463737 ?

Reply:
Hi Alida, I will update the reference, thank you. It is true that while mentioning that pulmonary alveolar type 1 are squamous cells, it is not the main focus on the paper and your proposed is more adequate. Cheers!